src/core_ocean/Registry.xml

```diff
@@ -2937,6 +2972,21 @@
 			 packages="frazilIce"
 		/>
 	</var_struct>
+	<var_struct name="timeVaryingForcing" time_levs="1">
```

Comment:
Here is another place where one or more packages would make sense, see the frazil examples above.